pages/math-sessions/math.js

```diff
@@ -0,0 +1,48 @@
+(async () => {
+
+  let airtable_res = await fetch(
+    `https://api.airtable.com/v0/appHwUzo4ARCQQlwr/Math%20Sessions?maxRecords=20&view=Grid%20view`,
+    {
+      method: "GET",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+        Authorization: `Bearer pat2bEq3dsaXHSBH9.2edd33a7b1c2de8fd5e4fe14b82900cf807d2c9b56dfead6a8bdd48715826409`,
+      },
+    }
+  );
```

Comment:
It is a good time to create either a single function for handling all airtable requests, or using global constants for things like API keys.